examples/getter-distortion/index.js

```diff
@@ -4,13 +4,16 @@ import createVirtualEnvironment from '@locker/near-membrane-dom';
 const { get } = Object.getOwnPropertyDescriptor(ShadowRoot.prototype, 'host');
 
 const distortionMap = new Map([
-    [get, () => {
-        console.error('forbidden');
-        return null;
-    }],
+    [
+        get,
+        () => {
+            console.error('forbidden');
+            return null;
+        },
+    ],
 ]);
 
-const evalScript = createVirtualEnvironment({ distortionMap });
+const evalScript = createVirtualEnvironment(window, { distortionMap, globalThis: window });
```

Comment:
same here,,, distortionMap is not longer good, and globalThis is the default behavior